Dockerfile

```diff
@@ -23,7 +23,6 @@ RUN npm config set registry http://registry.npmjs.org/ && \
     npm install -g -s flow-bin@0.60.0 && \
     yarn install --production=false && \
     yarn run lint && \
-    yarn run test && \
```

Comment:
I'm still figuring out why tests are failing on docker :( I hope to find a solution in 2-3 days.

Reply:
Sure, no rush, I disabled them now temporary. I hope you find a reason ;( I couldn't.

Reply:
https://github.com/verdaccio/verdaccio/commit/c2780d6746cd8115c4e1cc48f3a09c688b8d6fb9 
From the slack channel for the audience.
> "react does not throw errors on Production mode in Docker container we switch the environment to Production, to fix it, I just reset the env to something different and now they run fine"